scripts/build-armbian.sh

```diff
@@ -95,14 +95,15 @@ pushd "$BUILD_DIR" >/dev/null
   VENDOR="evcc" \
   VENDORURL="https://evcc.io" \
   IMAGE_SUFFIX="evcc-${RELEASE_NAME}" \
-  ./compile.sh \
+  ./compile.sh openwb \
     BOARD="$BOARD" \
     BRANCH=current \
-    RELEASE="bookworm" \
+    RELEASE="trixie" \
     BUILD_MINIMAL=no \
     BUILD_DESKTOP=no \
     KERNEL_CONFIGURE=no \
-    COMPRESS_OUTPUTIMAGE=sha,zip
+    COMPRESS_OUTPUTIMAGE=sha,zip \
+    INCLUDE_HOME_DIR=yes
```

Comment:
To auto-start firefox in kiosk mode, I need to create quite a few files in the /home/admin directory. So, instead of adding those to the first run script, I added this option to retain the state of the home directory as created during the build